app/src/main/java/com/example/bisit/ui/login/FindPasswordFragment.kt

```
@@ -0,0 +1,155 @@
+    private fun setupTextWatchers() {
+        // 1. 휴대폰 번호 포맷팅
+        val blockHyphenFilter = InputFilter { source, _, _, _, _, _ ->
+            if (source.toString() == "-") "" else null
+        }
+        val lengthFilter = InputFilter.LengthFilter(13)
+        binding.etPhone.filters = arrayOf(blockHyphenFilter, lengthFilter)
+        binding.etPhone.addTextChangedListener(PhoneNumberFormattingTextWatcher("KR"))
+
+        // 2. '번호 인증' 버튼 활성화
+        binding.etPhone.addTextChangedListener(object : TextWatcher {
+            override fun beforeTextChanged(s: CharSequence?, start: Int, count: Int, after: Int) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)